utility/mm2_upgrade-install-repo

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+
+"""
+This script changes the directory path from the fedora-install-X repositories
+for a given version.
+
+This can be used several time during the release cycle to move a repo from
+a beta state to a release state.
+
+This is currently kind of a work-around until the logic can be added into
+the UMDL script.
+
+TODO: test IRL
+"""
+
+import sys
+import re
+import os
+
+sys.path.insert(0, os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), '..'))
+import mirrormanager2.lib
+from mirrormanager2.lib.model import Repository
+from optparse import OptionParser
+
+
+# moving from pub/fedora/linux/releases/test/22_Beta/Server/x86_64/os/
+# to          pub/fedora/linux/releases/22/Server/x86_64/os
+# TODO: adjust the UMDL to do this
+
+
+def move_install_repo(session, version, test=False, debug=False):
+    product=mirrormanager2.lib.get_product_by_name(session, "Fedora")
+    ver = mirrormanager2.lib.get_version_by_name_version(
+            session, product.name, version)
+    if not ver:
+        print 'No version found for %s-%s' % (product.name, version)
+        return
+
+    for a in mirrormanager2.lib.get_arches(session):
+        if a.name == u'source':
+            continue
+
+        if options.version == 'development':
+            # We do not change development repos
+            continue
+        else:
```

Comment:
Since the line before this is a `continue`, is there no need for the `else:` block here?  You could perhaps unindent it and maybe get a little more readability.